forum/qa-plugin/tips-widget/tips-widget-page.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+class tips_widget_page
+{
+	function match_request($request)
+	{
+		if ($request == 'tips')
```

Comment:
niedostosowanie do PSR-2

Reply:
każdy if powinien mieć blok {} oraz warunki powinny mieć formę 'tips' === $request

Reply:
Poza tym ten kod mozna zmniejszyc do jednej linijki bez dwoch returnow.

Reply:
Owszem, mogłem wykorzystać operator ? :, ale ten zapis też jest odpowiedni. Nie widzę powodu na zmienianie jego oprócz względów estetycznych.

Reply:
Jest if, który nie dba o bezpieczeństwo typu danych i jego dokładność, są dwa returny, które zwiększają ilość wyjść z funkcji. To nie są pierdoły. To nie C++.